public/app/core/components/AppChrome/DockedMegaMenu/MegaMenuItem.tsx

```diff
@@ -121,7 +122,9 @@ export function MegaMenuItem({ link, activeItem, level = 0, onClick }: Props) {
                 />
               ))
           ) : (
-            <div className={styles.emptyMessage}>{link.emptyMessage}</div>
+            <div className={styles.emptyMessage} aria-live="polite">
+              {t('navigation.megamenu.starredEmptyMessage', 'Your starred dashboards will appear here')}
```

Comment:
hate to be the bearer of bad news, but this is already translated earlier in the process [here](https://github.com/grafana/grafana/blob/main/public/app/core/utils/navBarItem-translations.ts#L28) 😅

Reply:
Oh, interesting. I didn't know this file yet. 😅
I'll remove the second translation.